Makefile

```diff
@@ -284,6 +284,7 @@ integration-test:
 .PHONY: integration-test-docker
 integration-test-docker:
 	scripts/dev_scripts/integration_tests_docker.sh $(REPO_PATH) scripts/release_scripts/run_macaron.sh
+	scripts/dev_scripts/test_run_macaron_sh.py
```

Comment:
Instead of adding the run command here, can you please call this as a test from `integration_tests_docker.sh`?

Reply:
I have moved the command as you suggested. See 87bfcfec300f31e73f0a62b479c1206b4ddaecf4.